### DISCUSSION TITLE
Allowing UI elements outside of the game window when black bars surround it

### DISCUSSION THREAD

**Discussion by Gwiggle** [runelite/runelite#16344] [Ideas]
Example being in stretched fixed mode. Idea is to allow UI elements such as clue scroll helper or mahogany homes ui info in the black areas around the client to allow the actual game window to be less cluttered while keeping important information clearly visible

**Comment by SoNuclear** [comment]
Or simply even a sidebar tab for overlays that you would prefer to track but not have on screen. You could then reorder them, maybe even have a thicker sidebar with a customisable grid.

**Comment by raiyni** [comment]
Duplicate of https://github.com/runelite/runelite/issues/10414
